__tests__/models.test.js

```diff
@@ -263,6 +263,33 @@ describe("PATCH: /api/reviews/:review_id", () => {
   });
 });
 
+describe("DELETE: /api/comments/:comment_idgroup", () => {
+  it("should delete the given comment and return no content", () => {
```

Comment:
Could we double check this test is currently passing? As it looks like in your controller you are actually sending a full response body back not just an empty object?